roles/keystone-defaults/defaults/main.yml

```diff
@@ -83,6 +83,7 @@ keystone:
     sensu_checks:
       check_keystone_api:
         criticality: 'critical'
+    cron_bin_path:  "/sbin:/usr/sbin:{{ (os == 'rhel' ) | ternary('/usr/bin/', '/usr/local/bin/') }}"
```

Comment:
ok that makes sense since the default PATH has all of those defined in both RHEL and Ubuntu. I'll remove the PATH modification and try it out.